.devcontainer.json

```diff
@@ -1,68 +0,0 @@
-// Reference:
```

Comment:
Sadness?

Reply:
I'm not opposed too dev containers, but we don't use them. The sub-module links to a archived repo.